src/helpers.ts

```diff
@@ -179,6 +179,11 @@ export const deleteCookie = (name: string, path: string | undefined = "/") => {
 
 export const generateId = () => Math.random().toString(36).substring(2, 12);
 
+export const getUmExclusionState = () => {
+  let state = localStorage.getItem("um_exclusion")
+  return  state === undefined || state === null || state === "false" ? false : true
+};
+
```

Comment:
We are missing the isWindow function check. For example, in case of next.js the SDK will be breaking. Because, it won't be able to find the localStorage and will give an error in that case.

Reply:
I see. I'll test and do the needful accordingly as suggested.

Reply:
Updated